OpenIPSL/Electrical/Buses/Bus.mo

```diff
@@ -35,7 +35,7 @@ equation
           origin={0,-120},
           extent={{-60,-20},{60,20}},
           lineColor={170,0,0},
-          textString=DynamicSelect("Vpu", String(V, significantDigits=3))),Text(
+          textString=DynamicSelect("Vpu", String(v, significantDigits=3))),Text(
```

Comment:
I wonder, should we also change the default string from `"Vpu"` to simply `"v"` or perhaps even `"Voltage"` since we also have `"Angle"` below?